mkdocs.yml

```diff
@@ -1,7 +1,7 @@
 site_name: "Scality S3 CSI Driver"
 site_url: https://scality.github.io/mountpoint-s3-csi-driver
 repo_url: https://github.com/scality/mountpoint-s3-csi-driver
-edit_uri: edit/main/docs/            # ← branch that holds Markdown
+edit_uri: edit/documentation/S3CSI-64-add-core-documentation/docs/            # ← branch that holds Markdown
```

Comment:
```suggestion
edit_uri: edit/main/docs/            # ← branch that holds Markdown
```

Reply:
dropped the commit, thanks for the reminder.